client/src/components/common/request-details.vue

```diff
@@ -114,23 +109,9 @@ export default class RequestDetails extends Vue {
 <style lang="scss">
 @import "@/assets/scss/theme.scss";
 
-.choice-indicator {
-  background-color: $gray9;
-  color: white;
-  border-radius: 100%;
-  width: 1.75rem;
-  height: 1.75rem;
-  margin-right: 0.5rem;
-  margin-bottom: 5px;
-  box-sizing: border-box;
-  display: inline-flex;
-  justify-content: center;
-  align-items: center;
-  font-size: 0.8rem;
-  font-weight: bold;
-}
-
 h4 {
   font-weight: bold !important;
+  font-size: 16px;
+  color: $gray9;
```

Comment:
Please change font-size to 1rem.

Please change color to "$dk-text".

(This will all help simplify/reduce the CSS when/if we get around to it.)